project-addons/crm_claim_rma/wizard/equivalent_products_wizard.py

```diff
@@ -20,51 +19,42 @@
 ##############################################################################
 
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
-class equivalent_products_wizard(models.TransientModel):
+class EquivalentProductsWizard(models.TransientModel):
 
     _name = "equivalent.products.wizard"
     _description = "Wizard for change products in claim."
 
     product_tag_ids = fields.Many2many("product.tag",
-                                    "product_tag_wzf_equivalent_rel",
-                                    "wizard_id", "tag_id", "Tags")
+                                       "product_tag_wzf_equivalent_rel",
+                                       "wizard_id", "tag_id", "Tags")
     product_id = fields.Many2one('product.product', 'Product selected')
     line_id = fields.Many2one('claim.line', 'Line')
     real_stock = fields.Float("Real Stock", readonly=True)
     virtual_stock = fields.Float("Virtual Stock", readonly=True)
 
     def default_get(self, cr, uid, fields, context=None):
-        res = super(equivalent_products_wizard, self).default_get(cr, uid, fields, context=context)
+        res = super(EquivalentProductsWizard, self).default_get(cr, uid, fields, context=context)
         if context.get('line_id'):
             claim_line_id = self.pool.get('claim.line').browse(cr, uid, context['line_id'])
             res['product_id'] = claim_line_id.product_id.id
             res['real_stock'] = claim_line_id.product_id.qty_available
             res['virtual_stock'] = claim_line_id.product_id.virtual_available
-            res['product_tag_ids'] = \
-                [(6, 0, claim_line_id.product_id.tag_ids.ids)]
+            res['product_tag_ids'] = [(6, 0, claim_line_id.product_id.tag_ids.ids)]
         return res
 
-    def onchange_product_id(self, cr, uid, ids, product_id, context=None):
-        if not product_id:
-            return {}
-        prod_obj = self.pool.get('product.product')
-        prod_id = prod_obj.browse(cr, uid, product_id)
-        virtual_stock = prod_id.virtual_available
-        real_stock = prod_id.qty_available
-        return {
-            'value': {'virtual_stock': virtual_stock,
-                      'real_stock': real_stock}
-        }
+    @api.onchange('product_id')
+    @api.multi
+    def onchange_product_id(self):
+        for prod in self:
+            self.virtual_stock = prod.virtual_available
```

Comment:
prod no self, aunque en un on_change no debería de venir nunca más d eun id, por lo que no debería der necesario le for